lib/DDG/Goodie/DaysBetween.pm

```diff
@@ -13,19 +13,18 @@ zci is_cached => 0;
 zci answer_type => "days_between";
 
 my $datestring_regex = datestring_regex();
-my @monhs  = qw( Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec );
+my @months  = qw( Jan Feb Mar Apr May Jun Jul Aug Sep Oct Nov Dec );
 
-handle query_lc => sub {
-    return unless ($_ =~ qr/(days since)(?: *)($datestring_regex)/i) || ($_ =~ qr/($datestring_regex) (?:(?:and|to) )?($datestring_regex)(?:[,]? inclusive)?$/i);
```

Comment:
That's true, although if users search for `days between 12 July 2016` and receive an answer for `days between 12 July 2016 and today`, I don't think that's a bad thing. Their initial query makes no sense but the search engine gives them an answer that does, and they could correct their query if they want. This way should also handles "from" nicely, so that it doesn't matter if the user enters a second date or not.

However, I'll leave it to you to decide. If you prefer to use `query_lc` rather than `remainder`, then maybe another approach could be to check for the word "since" at the start of the `handle` section, then remove the trigger words parse with either of the regexes depending on whether "since" existed in the query. This would allow you to keep the `^` in the regex which is what was making the final test pass.